reference/release-notes/release-notes-template.md

```diff
@@ -3,9 +3,15 @@ orphan: true
 ---
 # < Release version >
```

Comment:
The release notes template is updated to align with the latest release notes template that will be announced as a standard across Canonical products. 

The 1.23.1 release notes is created with this new template.